### PR TITLE
Refactor sidebar

### PR DESCRIPTION
**Pull request by b-onc**
Our Sidebar is much closer to other SDKs now, and looks much better

Quickly check it out here: https://swift-docs-new-sidebar.loca.lt/chat/docs/sdk/ios/?language=swift

### CODE REVIEW

File: docusaurus/sidebars-ios.json [hunk 1]
```diff
@@ -1,14 +1,18 @@
 {
   "docs": [
-    "introduction",
     {
-      "Getting Started": [
-        "getting-started/integration",
-        "getting-started/up-running"
-     ]
+      "Basics": [
```
Comment (DominikBucher12) [inline docusaurus/sidebars-ios.json:4]: 👍

**Comment by nuno-vieira** [comment]
image

Can we just call it "Controllers" ? Instead of "StreamChat Controller". IMO looks better. Unless we instead of "UI Components" have "StreamChatUI Components"

**Comment by b-onc** [comment]
@nuno-vieira good idea 👍

File: docusaurus/sidebars-ios.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
       ]
     },
     {
-      "StreamChat Controllers": [
+      "Controllers": [
```
Comment (DominikBucher12) [inline docusaurus/sidebars-ios.json:45]: 👍

**Review by DominikBucher12 — APPROVED** [review]
From my side this looks good, I like this actually better... Aand +1 point For the migration guide outside of scope of name of PR :D

**Pull request merged**